### ISSUE TITLE
add current (and possibly backlogged) numbers in plain text.

### ISSUE THREAD

**Issue by lilmike** [munin-monitoring/munin#503]
I'm blind, so use a screen reader to access websites... which obviously does not read complex things such as graphs for me. While my main purpose of munin was to alert me if something went wrong, it would be nice if I could see the actual numbers of the last (and possibly backlogged) runs of munin in the web pages instead of having to ssh into each server. I don't think it would be that hard to build, but if you need any testing let me know.
-Michael.

**Comment by ssm** [comment]
The version of munin currently in development can show the graph data as
structured text (json or xml) as well as images.  Expected release date
is late august.

From this release, it should be possible to make a text-based interface.

-- 
Stig Sandbeck Mathisen
